spring-cloud-alibaba-examples/seata-example/account-service/src/main/resources/application.yml

```diff
@@ -0,0 +1,58 @@
+base:
+  config:
+    mdb:
+      hostname: 127.0.0.1
+      dbname: seata
+      port: 3306
+      username: root
+      password: 123456
+
+server:
+  port: 18084
+
+spring:
+  cloud:
+    nacos:
+      discovery:
+        server-addr: 127.0.0.1:8848
+  application:
+    name: account-service
+  main:
+    allow-bean-definition-overriding: true
+  datasource:
+    name: storageDataSource
+    type: com.alibaba.druid.pool.DruidDataSource
+    driver-class-name: com.mysql.cj.jdbc.Driver
+    url: jdbc:mysql://${base.config.mdb.hostname}:${base.config.mdb.port}/${base.config.mdb.dbname}?useUnicode=true&characterEncoding=UTF-8&serverTimezone=UTC&zeroDateTimeBehavior=convertToNull
```

Comment:
By the way, I can't run the example in my environment, the error is below:
---------
com.mysql.cj.jdbc.exceptions.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.

Reply:
MySQL database service started? Is the database connection address configured?